board/arm-64bit-toolchain.cmake

```diff
@@ -0,0 +1,55 @@
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR ARM)
+
+if(MINGW OR CYGWIN OR WIN32)
+    set(UTIL_SEARCH_CMD where)
+elseif(UNIX OR APPLE)
+    set(UTIL_SEARCH_CMD which)
+endif()
+
+## You need to add the full path in TOOLCHAIN_PREFIX, if you do not use the
+## standard gcc toolchain 
+set(TOOLCHAIN_PREFIX aarch64-linux-gnu-)
+
+## In some distribution, a dynamic link for aarch64-linux-gnu-gcc may not be
+## found or created, instead it might be labelled with the version at the end
+## For instance: aarch64-linux-gnu-gcc-5
+## Therefore, if dynamic link exists, you do not have to specify the version
+set(VERSION_NUMBER "" CACHE STRING "Enter the version number of the compiler")
+
+execute_process(
+  COMMAND ${UTIL_SEARCH_CMD} ${TOOLCHAIN_PREFIX}gcc${VERSION_NUMBER}
+  OUTPUT_VARIABLE BINUTILS_PATH
+  OUTPUT_STRIP_TRAILING_WHITESPACE
+)
```

Comment:
It looks like `BINUTILS_PATH` is never used?

Reply:
I will check :+1: